.github/workflows/release.yml

```diff
@@ -9,17 +9,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.12
+    - name: Set up Go 1.13.3
```

Comment:
Latest stable is 1.13.4

Reply:
That sounds right, I am actually fully aware of this as homebrew-core maintainer 😄